test/postgres/setup-postgres.sh

```diff
@@ -18,6 +18,8 @@ ssl_key_file = '/share/secrets/postgres.key'
 ssl_ca_file = '/share/secrets/ca.crt'
 hba_file = '/share/conf/pg_hba.conf'
 wal_level = logical
+fsync = off
```

Comment:
Please run Nightly to confirm that this change is safe on all of our chaos workloads.